pyproject.toml

```
@@ -1,20 +1,15 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "finch-tensor"
 version = "0.1.3"
 description = ""
-authors = ["Willow Ahrens <willow.marie.ahrens@gmail.com>"]
+maintainers = [{ name = "Willow Ahrens", email = "willow.marie.ahrens@gmail.com" }]
 readme = "README.md"
-packages = [{include = "finch", from = "src"}]
-
-[tool.poetry.dependencies]
-python = "^3.8"
-juliapkg = "^0.1.10"
-juliacall = "^0.9.15"
+requires-python = ">=3.8"
+dependencies = ["juliapkg>=0.1.10", "juliacall>=0.9.15", "semantic-version>=2.10.0"]
```

```suggestion
dependencies = ["juliapkg>=0.1.10", "juliacall>=0.9.15"]
```

Already handled by `setuptools_scm` I think.

---

Addressed in #2